bundles/org.openhab.binding.robonect/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -45,6 +45,10 @@
 
 			<channel id="health-temperature" typeId="temperatureType"/>
 			<channel id="health-humidity" typeId="humidityType"/>
+
+			<channel id="blades-quality" typeId="bladesQualityType"/>
+			<channel id="blades-days" typeId="bladesQualityDaysType"/>
+			<channel id="blades-hours" typeId="bladesQualityHoursType"/>
```

Comment:
When reading the channel ids, one can easily get confused, because they differ only in time unit (days vs. hours), but without mentioning their meaning (usage vs. last replacement). Perhaps it would be better to drop the unit and instead name the channels after their meaning?